Zilon.Core/Zilon.Core/Commands/Sector/AttackCommand.cs

```diff
@@ -130,15 +135,23 @@ protected override void ExecuteTacticCommand()
             var propResources = new List<Resource>();
             foreach (var prop in props)
             {
-                var propResource = prop as Resource;
-                if (propResource == null)
+                switch (prop)
```

Comment:
Codacy found an issue: [Replace this 'switch' statement with 'if' statements to increase readability.](https://app.codacy.com/manual/kreghek/Zilon_Roguelike/pullRequest?prid=6105716)